### PR TITLE
Add Fiiish!-RS

### PR DESCRIPTION
**Pull request by AndreasOM**

### CODE REVIEW

File: content/games/data.toml [hunk 1]
```diff
@@ -157,6 +157,14 @@ categories = ["simulation"]
 repository_url = "//github.com/amethyst/evoli/"
 image = "/assets/img/evoli-prealpha.gif"
 
+[[items]]
+name = "Fiiish! RS"
+description = "A simple remake of the Fiiish endless swimmer in Rust."
+categories = ["action", "released"]
```
Comment (ozkriff) [inline content/games/data.toml:163]: I'm not sure the project belongs to the "released" category atm:

> Finished games that have been released

The game's page says it's "still in early development."

Comment (AndreasOM) [reply content/games/data.toml:163]: The games page is wrong. 🤦  Will update once I get out of this train.
The rust version of the game has parity with the iOS version.

Comment (ozkriff) [reply content/games/data.toml:163]: :ok_hand:

**Review by ozkriff — APPROVED** [review]

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by ozkriff** [comment]
Tnx for the PR!

**Pull request merged**